src/Sidebar/ui/SidebarRoot.tsx

```diff
@@ -276,54 +275,54 @@ export function SidebarRoot(props: IProps): React.ReactElement {
     function handleShortcuts(this: Document, event: KeyboardEvent): any {
       if (Settings.DisableHotkeys) return;
       if ((props.player.isWorking && props.player.focus) || redPillFlag) return;
-      if (event.keyCode == KEY.T && event.altKey) {
+      if (event.key === "t" && event.altKey) {
```

Comment:
how about instead we change the KEY constants to be these static strings, there are other devs that have a pet peeve about magic strings

Reply:
Added the GIF and changed it to the KEY constant.